src/app.ts

```diff
@@ -22,6 +25,47 @@ app.use("/", bodyParser.json());
 
 app.use("/auth", authRouter);
 
+// Create a new attendee
```

Comment:
move all this code into an `attendee-router.ts` file inside of `services/attendees`!